File: clientssr/src/app/Model/Hadith/hadith-contant-response.ts
import { HadithListResponse } from "./hadith-list-response";

export interface HadithContantResponse {
  classificationId: number;
  classificationName?: string | null;
  bookId: number;
  bookName?: string | null;
  babId: number;
  babName?: string | null;
  hadiths: HadithListResponse[];
}

File: Application/CQRS/Hadiths/Request-Response/HadithContantResponse.cs
namespace Application.CQRS.Hadiths
{
    public class HadithContantResponse
    {
        public int ClassificationId { get; set; }
        public string? ClassificationName { get; set; }
        public int BookId { get; set; }
        public string? BookName { get; set; }
        public int BabId { get; set; }
        public string? BabName { get; set; }
        public List<HadithListResponse> Hadiths { get; set; } = new List<HadithListResponse>();
    }
}
